plugins/targetlocker/dblocker/dblocker.go

```diff
@@ -294,8 +291,8 @@ func (d *DBLocker) TryLock(jobID types.JobID, targets []*target.Target, limit ui
 	if len(targets) == 0 || limit == 0 {
 		return nil, nil
 	}
-
-	return d.handleLock(int64(jobID), targetIDList(targets), limit, d.lockTimeout, true)
+	lockTimeout := jobTargetManagerAcquireTimeout + config.LockRefreshTimeout
```

Comment:
Same

Reply:
Reverted this, it was probably an experiment leftover.